apps/menu-bar/src/popover/Core.tsx

```diff
@@ -154,22 +154,22 @@ function Core(props: Props) {
 
         if (!localFilePath) {
           setStatus(Status.DOWNLOADING);
-          const [buildPath] = await Promise.all([
-            downloadBuildAsync(appURI, setProgress),
-            ensureDeviceIsRunning(device),
-          ]);
-
+          const buildPath = await downloadBuildAsync(appURI, setProgress);
```

Comment:
any reason for not doing this in in parallel?

Reply:
I was attempting to debug download issues, and also the ensureDeviceIsRunning call seemed semantically related to install and not download...